pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -573,38 +625,38 @@ private static int trueCount(final Boolean... b) {
     }
   }
 
-  private File getNodePrivateKeyFile() {
+  private File getNodePrivateKeyFile(@Nullable final File nodePrivateKeyFile) {
```

Comment:
Pantheon doesn't currently use `@Nullable` so better to leave it off for consistency.